src/rpc/rawtransaction.cpp

```diff
@@ -95,6 +95,35 @@ void TxToJSON(const CTransaction& tx, const uint256 hashBlock, UniValue& entry)
     }
 }
 
+std::string getSaplingTxHelpInfo()
+{
+    return "  \"valueBalance\" : n,          (numeric) The net value of spend transfers minus output transfers\n"
```

Comment:
nit: shouldn't have whitespace before the `:` character. We are very inconsistent on this in the help output and something i'm looking to rectify, but would be good to not introduce new code with formatting issues.